dbt_platform_helper/providers/version.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC
+
+import requests
+
+from dbt_platform_helper.providers.semantic_version import SemanticVersion
+
+
+class VersionProvider(ABC):
+    pass
```

Comment:
I started writing this suggestion, then realised that the two concrete implementations methods have different arguments. So I'm wondering what the benefit of the abstract class is?
```suggestion
class VersionProvider(ABC):
    @abstractmethod
    def get_latest_version(repo_name: str, tags: bool = False) -> SemanticVersion:
        pass
```

Reply:
I wonder whether the interface can be the same if `GithubVersionProvider. get_latest_version()` returns whichever version is highest from the releases or tags?